src/main/java/org/broadinstitute/hellbender/utils/hmm/segmentation/HiddenMarkovModelPostProcessor.java

```diff
@@ -308,8 +309,9 @@ private void performSegmentation() {
      */
     private Map<String, List<HiddenStateSegment<S, T>>> calculateBestPathSegments() {
         final Map<String, List<HiddenStateSegment<S, T>>> allSegments = new LinkedHashMap<>(sampleNames.size());
-        IntStream.range(0, numSamples)
+        IntStream.range(0, numSamples).parallel()
```

Comment:
Perhaps because overhead varies (and can actually degrade performance), we've defaulted to not using them.